Update issue templates (#341)
plotly/dash-bio: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve

Reviewer: ```suggestion
about: Report a bug to help us improve Dash Bio
```

title: ''
labels: ''
assignees: ''

---

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**

> Steps to reproduce the behavior:

Reviewer: How about 'starting' a list with `-` (or `*`) on the next line, like you did line 25?

Author: Good catch! Fixed in ff7c72f.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**

> If applicable, add screenshots to help explain your problem.

Reviewer: ```suggestion
If applicable, add screenshots to help explain the issue.
```
